### PR TITLE
Update github actions

### PR DESCRIPTION
**Pull request by dhyanio**
# Description

- Add. readme script
- Add, push script

### CODE REVIEW

File: src/push.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+function push {
+ 
+ 
+  git config --global user.email "ajay@clouddrove.com"
```
Comment (sohanyadav) [inline src/push.sh]: why ajay shoud be anmol

File: src/push.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  git config --global user.name "Ajay dhyani"
+  git add . && git commit -m "upload"
```
Comment (sohanyadav) [inline src/push.sh]: commit should be make create read me

File: src/terratest_go.sh [hunk 3]
```diff
@@ -3,13 +3,14 @@
...
   echo "install the terratest dependencies"
-  apk add -d --update --no-cache go gcc build-base
+  apk add -d --update --no-cache go curl gcc build-base
 
   echo "Install Go for terratest"
   if [ -x /github/home/.go ]; then
     echo exists
 else
   echo "Install Go for terratest"
+<<<<<<< HEAD
```
Comment (sohanyadav) [inline src/terratest_go.sh]: ???????/?

File: src/terratest_go.sh [hunk 4]
```diff
@@ -21,6 +22,32 @@ apk del .build-deps
 go version
 
   echo "Install Go package fo terratest"
+=======
```
Comment (sohanyadav) [inline src/terratest_go.sh]: ???????????

**Review by sohanyadav — CHANGES_REQUESTED** [review]
check

File: src/terratest_go.sh [hunk 5]
```diff
@@ -21,6 +22,32 @@ apk del .build-deps
...
+  mkdir bin
+  source /etc/profile
+  cd /usr/local/go/src
+  echo 123
+  ./make.bash
+  go version
+  echo "Install dep"
+  curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
+  echo "Install Go package for terratest"
+>>>>>>> bebc31ca55bce4dd0e77022d5491a10630cb8076
```
Comment (harmanjyotkaur) [inline src/terratest_go.sh]: why we have this here ? please fix it.

**Review by harmanjyotkaur — CHANGES_REQUESTED** [review]

**Review by sohanyadav — APPROVED** [review]
LGTM!!

**Pull request merged**